packages/flutter_test/test/widget_tester_test.dart

```diff
@@ -309,6 +309,28 @@ void main() {
       expect(count, 6);
     });
 
+    testWidgets('pumpContinuous', (WidgetTester tester) async {
+      await tester.pumpWidget(const Text('foo', textDirection: TextDirection.ltr));
+      int count;
+
+      final AnimationController test = AnimationController(
+        duration: const Duration(milliseconds: 200),
+        vsync: tester,
+      );
+      count = await tester.pumpContinuous(const Duration(milliseconds: 80));
+      expect(count, 1); // No animation, it always pumps at least one frame
+
+      test.forward(from: 0.0);
+      count = await tester.pumpContinuous(const Duration(milliseconds: 80));
+      expect(count, 1 + (80/16.685).floor()); // ends after 80 ms
```

Comment:
The 16.685 seems to be from our default double frequency = 59.94. Maybe it's clearer to double frequency = kDefaultFrequency and use 1000 / kDefaultFrequency here?